docs/core/install/includes/package-manager-heading-hack-pkgname.md

```diff
@@ -26,6 +26,6 @@ The version of the SDK or runtime to install. This article will always give the
 - Install the ASP.NET Core 3.1 runtime: `aspnetcore-runtime-3.1`
 - Install the .NET Core 2.1 runtime: `dotnet-runtime-2.1`
 
-### Troubleshoot
+### Combination failed
```

Comment:
"Combination" seems weird here to me, it makes me think of a combination of .NET Core, SDK, and ASP.NET Core packages not working, but it's just talking about one package name. This extends to `package combination` below.

Something like "Installing the custom package name fails" seems clearer to me.